### PR TITLE
Make ignore-messages can be specified by regex

### PR DESCRIPTION
**Pull request by nowsprinting**
### Changes
Make "Ignore Messages" can be specified by regex.

### Priority
ASAP

---

### Contribution License Agreement

- [x] By placing an "x" in the box, I hereby understand, accept and agree to be bound by the terms and conditions of the [Contribution License Agreement](https://dena.github.io/cla/).

### CODE REVIEW

File: Tests/Runtime/Utilities/LogMessageHandlerTest.cs [hunk 1]
```diff
@@ -152,6 +152,36 @@ public async Task HandleLog_ContainsIgnoreMessage_notReported()
...
+            var sut = new LogMessageHandler(settings, spyReporter);
+
+            settings.ignoreMessages = new[] { "ignore.+ignore" };
+            settings.handleException = true;
+            sut.HandleLog("ignore", string.Empty, LogType.Exception);
+            await UniTask.NextFrame();
+
+            Assert.That(spyReporter.Arguments, Is.Not.Empty);
+        }
+
```
Comment (asurato) [inline Tests/Runtime/Utilities/LogMessageHandlerTest.cs:184]: must

There isn't a test case when invalid regex are entered such as "[a".
Add the test case or tell me how it behaves in the case, please.

**Review by asurato — CHANGES_REQUESTED** [review]
Hi, I'm in SWET.
I looked over the changes, and I commented.

File: README.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ Set up a filter to catch abnormal log messages and notify using Reporter.
   <dt>Handle Error</dt><dd>Report when error message is detected in log</dd>
   <dt>Handle Assert</dt><dd>Report when assert message is detected in log</dd>
   <dt>Handle Warning</dt><dd>Report when warning message is detected in log</dd>
-  <dt>Ignore Messages</dt><dd>Log messages containing this string will not be report</dd>
+  <dt>Ignore Messages</dt><dd>Log messages containing this string will not be report. Can specify regex.</dd>
```
[the remark this answers is not in the document]
Comment (nowsprinting) [reply README.md]: Thanks, fixed in 4877ff22bd7514d13cd79d7fa62b2aee48a57292.

File: Tests/Runtime/Utilities/LogMessageHandlerTest.cs (hunk 1 above)
Comment (nowsprinting) [reply Tests/Runtime/Utilities/LogMessageHandlerTest.cs:185]: Fixed in 4ee60f362c20ab7e7740b93ede8a1ddbe9d6daef.

1. Add catch exception in `LogMessageHandler`, Do not throw an exception, only output to the logger.
2. Add test case

**Comment by nowsprinting** [comment]
@asurato san, Thank you for your review!
I've fixed the recommendation, so please review it again!

**Review by asurato — APPROVED** [review]
LGTM

**Review by bo40 — APPROVED** [review]
LGTM

**Pull request merged**